README.md

```diff
@@ -8,7 +8,7 @@ CollabVM was coded and written by Cosmic Sans, Dartz, Geodude, and modeco80... A
 ### License
 CollabVM Server is licensed under the [GNU Public License 3](https://www.gnu.org/licenses/gpl-3.0.en.html).
 
-The CollabVM Server Plugin Interface "CVMpi" is licensed under the [Lesser GNU Public License 3](https://www.gnu.org/licenses/lgpl-3.0.en.html).
+The CollabVM Server Plugin Interface "CVMpi" is licensed under the [Lesser GNU Public License 3](https://www.gnu.org/licenses/lgpl-3.0.en.html). 
```

Comment:
I would reword this because the plugin interface isn't like. completely seperate from the server? the server source comes with and should optionally install sample plugin sources, something sorta like:

```diff
+ CollabVM Server is dual-licensed under the [GNU Public License 3](https://www.gnu.org/licenses/gpl-3.0.en.html) for the server, and the [LGPLv3](https://www.gnu.org/licenses/lgpl-3.0.en.html) license for the plugin interface and provided sample plugin code.
```

Reply:
i'll look into that.

Reply:
done https://github.com/yellows111/collab-vm-server/commit/b066e1632614be9ff484e36ce16d10cf9bfc4aee